x-pack/plugins/security_solution/public/flyout/entity_details/components/user_details_body.tsx

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiSpacer,
+  EuiHorizontalRule,
+  EuiIcon,
+  EuiBadge,
+  EuiText,
+  EuiFlexItem,
+  EuiFlexGroup,
+  useEuiTheme,
+  EuiTitle,
+  EuiPanel,
+} from '@elastic/eui';
+
+import React, { useMemo } from 'react';
+import { css } from '@emotion/react';
+import { max } from 'lodash/fp';
+import { SecurityPageName } from '@kbn/security-solution-navigation';
+import { getUsersDetailsUrl } from '../../../common/components/link_to/redirect_to_users';
+import type {
+  ManagedUserData,
+  ObservedUserData,
+} from '../../../timelines/components/side_panel/new_user_detail/types';
+import { ManagedUser } from '../../../timelines/components/side_panel/new_user_detail/managed_user';
+import { ObservedUser } from '../../../timelines/components/side_panel/new_user_detail/observed_user';
+import * as i18n from './translations';
+
+import type { RiskScoreEntity } from '../../../../common/search_strategy';
+import { SecuritySolutionLinkAnchor } from '../../../common/components/links';
+import type { RiskScoreState } from '../../../explore/containers/risk_score';
+import { PreferenceFormattedDate } from '../../../common/components/formatted_date';
+import { RiskSummary } from './risk_summary';
+
+export const QUERY_ID = 'usersDetailsQuery';
+
+interface UserDetailsBodyComponentProps {
+  userName: string;
+  observedUser: ObservedUserData;
+  managedUser: ManagedUserData;
+  riskScoreState: RiskScoreState<RiskScoreEntity.user>;
+  contextID: string;
+  scopeId: string;
+  isDraggable: boolean;
+}
+
+export const UserDetailsBody = ({
+  userName,
+  observedUser,
+  managedUser,
+  riskScoreState,
+  contextID,
+  scopeId,
+  isDraggable,
+}: UserDetailsBodyComponentProps) => {
+  const { euiTheme } = useEuiTheme();
+
+  const lastSeenDate = useMemo(
+    () =>
+      max([observedUser.lastSeen, managedUser.lastSeen].map((el) => el.date && new Date(el.date))),
+    [managedUser.lastSeen, observedUser.lastSeen]
+  );
+
+  return (
+    <>
+      <EuiPanel hasShadow={false}>
```

Comment:
using the [FlyoutBody](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/flyout/shared/components/flyout_body.tsx) component in the parent will allow you to remove all the `EuiPanel` in this file.

Reply:
please check this comment: https://github.com/elastic/kibana/pull/169514#discussion_r1393825123

Reply:
yeah I understand. See my answer to that comment, I feel like we should use the code was written in the shared folder and have consistency between all the flyouts. Like I said, when I discussed this with UIUX months ago, they were fine with the change, and they've seen the alert details flyout for months now and nobody complained 😆

Reply:
It makes sense to me. I [updated](https://github.com/elastic/kibana/pull/169514/commits/40bf6687ac27e68ff2df0d49b1de8f5549a45c39) the component to use `FlyoutBody`.
Thank you for the detailed explanation. 💙